sdk/identity/identity/samples-dev/workloadIdentityCredential.ts

```diff
@@ -16,9 +16,9 @@ async function testDefaultCredential() {
 
 async function testWorkloadCredential() {
   const credential = new WorkloadIdentityCredential({
-    tenantId: process.env.AZURE_TENANT_ID!,
-    clientId: process.env.AZURE_CLIENT_ID!,
-    federatedTokenFilePath: process.env.AZURE_FEDERATED_TOKEN_FILE!,
+    tenantId: process.env.AZURE_TENANT_ID,
```

Comment:
Is it necessary to specify them all here or just let the environment do the work?

Reply:
I wanted to provide an example with passing the parameters.